.github/workflows/release-develop.yml

```diff
@@ -0,0 +1,52 @@
+name: Budibase Release
+
+on: 
+ push:
+    branches:
+      - develop
+
+env:
+  POSTHOG_TOKEN: ${{ secrets.POSTHOG_TOKEN }}
+  POSTHOG_URL: ${{ secrets.POSTHOG_URL }}
+  SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
+      
+jobs:
+  release:
+    runs-on: ubuntu-latest 
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - run: yarn 
+      - run: yarn bootstrap 
+      - run: yarn lint 
+      - run: yarn build 
+      - run: yarn test
+
+      - name: Configure AWS Credentials
```

Comment:
Not sure if this is needed?

Reply:
Not sure if its needed either, we release to Docker hub rather than ECR, so I think the AWS creds are kind of pointless now.

Reply:
That's what I was thinking - I'll make a note to revisit it for both the master and develop release and leave it in for now so they are at least consistent